salt/modules/reg.py

```diff
@@ -33,6 +33,11 @@
 # Import Salt libs
 import salt.utils.platform
 from salt.exceptions import CommandExecutionError
+try:
+    import salt.utils.win_reg
+    HAS_WINDOWS_MODUELS = salt.utils.win_reg.HAS_WINDOWS_MODULES
```

Comment:
Should probably be `HAS_WINDOWS_MODULES`... unless we're dueling...

Reply:
@twangboy good catch, thanks!